### ISSUE TITLE
Question about dataset split

### ISSUE THREAD

**Issue by neolifer** [CRIPAC-DIG/GRACE#4]
Hi,
Is there any reason that you use random split instead of the public split? I can't get similar performance in Cora public split compared to in random split based on the same hyperparameters setting (80.5 vs 83.3 in accuracy ).

**Comment by SXKDZ** [comment]
Hello,

We choose to randomly sample training/test sets in honoring to a pure unsupervised setting. The amount of training samples in the two cases is different. BTW, it is generally not encouraged to evaluate graph models on these datasets.  You may refer to our [newly released benchmarking study](https://openreview.net/forum?id=UuUbIYnHKO) for results on larger-scale datasets.
